codecov.yml

```diff
@@ -1,2 +1,4 @@
 ignore:
-  - "**/test/*"
+  - "**/test/**/*"
+  - "CMR/python/docit.py"
+  - "CMR/python/setup.py"
```

Comment:
Verified this gives us the results we want via dashboard: https://codecov.io/gh/nasa/eo-metadata-tools/tree/98544f458764df9d12a3579b92882ea587f91bd0